.github/workflows/main.yml

```diff
@@ -24,10 +24,12 @@ jobs:
       - name: Widget Tests
         run: melos run test:widget
       - name: Build release package
-        run: flutter build apk --release
+        run: |
+          flutter build apk --release
+          mv build/app/outputs/flutter-apk/app-release.apk build/app/outputs/flutter-apk/app-release-${{ github.event.number }}.apk
```

Comment:
do we need to rename the apk or do we just change the name parameter below? this gets deleted after a week so we are not creating storage that needs to be named properly right? this approach would require only changing the name parameter in line 33. @bakicelebi opinion?

Reply:
Yeah, but when we download the apk we get a zip file containing `app-release.apk`. So I think we should rename that also to `app-release-{{PR number}}.apk` as we'll be extracting that only.
Is it so @juliajulie95 ma'am?

Reply:
Yeah than it is better I think